great_expectations/data_context/data_context/cloud_data_context.py

```diff
@@ -142,7 +142,24 @@ def get_ge_cloud_config(
     ) -> GeCloudConfig:
         """
         Build a GeCloudConfig object. Config attributes are collected from any combination of args passed in at
-        runtime, environment variables, or a global great_expectations.conf file (in order of precedence)
+        runtime, environment variables, or a global great_expectations.conf file (in order of precedence).
```

Comment:
@anthonyburdi I deleted the method stub you wrote in the migrator but moved the docstring here since I thought it was quite good 😄